src/components/Messages/CreateMessageModal/CreateMessageModal.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  IconButton,
+  Typography,
+  Box,
+} from "@mui/material/";
+import CloseIcon from "@mui/icons-material/Close";
+import SearchBarMessages from "./SearchBarMessages";
+import MessagesModalList from "./MessagesModalList";
+import { useState } from "react";
+
+const styles = {
+  dialog: {
+    height: "60%",
+    borderRadius: 4,
+  },
+  dialogContent: {
+    padding: 0,
+  },
+  dialogTitle: {
+    paddingBottom: 0,
+    paddingLeft: 0.5,
+    paddingRight: 0.5,
+    paddingTop: 1,
+    display: "flex",
+    alignItems: "center",
+  },
+  titleBox: { paddingLeft: 3, width: "100%" },
+  headerTitle: {
+    fontWeight: "bold",
+  },
+  modalButtonBox: { paddingRight: 2 },
+  modalButton: { borderRadius: 10 },
+  modalHeader: {
+    display: "flex",
+    width: "100%",
+    alignItems: "center",
+    justifyContent: "space-between",
+  },
+};
+
+type CreateMessageModalProps = {
+  onClose: () => void;
+  open: boolean;
+};
+
+export default function CreateMessageModal({
+  onClose,
+  open,
+}: CreateMessageModalProps) {
+  const [focusSearchBar, setFocusSearchBar] = useState(false);
+
+  return (
+    <Dialog
+      fullWidth
+      onClose={onClose}
+      open={open}
+      scroll="paper"
+      PaperProps={{ sx: styles.dialog }}
+    >
+      <DialogTitle sx={styles.dialogTitle}>
+        <IconButton onClick={onClose}>
+          <CloseIcon />
+        </IconButton>
+        <Box sx={styles.modalHeader}>
```

Comment:
Does this div do anything? If not can we remove it and the corresponding style?

Reply:
Nope, outdated styling